app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command, old_file, new_file = command.split()
```

Comment:
what if command split does not return 3 elements as you expect? 
Also from checklist:

> Make sure you check that the cp input command is correct.